src/bin/fxrunner/main.rs

```diff
@@ -0,0 +1,50 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+mod config;
+
+use std::path::PathBuf;
+use std::process::exit;
+
+use slog::{error, info, Logger};
+use snafu::{ResultExt, Snafu};
+use structopt::StructOpt;
+
+use crate::config::Config;
```

Comment:
This is probably obvious to Rustaceans, but it might be good to say that this points to the mod's `Config` and *not* [this](https://docs.rs/config/0.9.3/config/struct.Config.html).

Same would apply to fxrecorder, too.

Reply:
In Rust 2018, things prefixed by `crate` are scoped to the local crate. If it was from the `config` crate, it would be `config::Config` (and would be grouped with other non-crate imports above).

Reply:
Again, *I* know that but I was hoping we could make it explicit for "future" me.